scpred_train_test_split.R

```diff
@@ -84,6 +84,10 @@ set.seed(opt$random_seed)
 print(opt$input_sce_object)
 # preprocess data from SCE object 
 sce_object = readRDS(opt$input_sce_object)
+# check for 'normcounts' slot
```

Comment:
This sort of silent correction always makes me nervous. An object having 'normcounts' has meaning- i.e. it means that the counts have been normalised. A user could pass their un-normalised counts in by mistake. The better thing to do is to add an 'assay' parameter to the script with help text like:

"name of assay in the SCE object containing normalised counts, 'normcounts' by default". 

That way, the default behaviour of reminding the user to normalise persists, but we can override if we know the counts we're passing are normalised.

Of course we could also fix our DropletUtils wrapper to have a parameter about the assay name to use.

Reply:
I think then we'll need to modify the DropletUtils package (I can make a fork). The reason is that when the script `read10xCounts.R` initialises an SCE object, it puts 'counts' into the assays slot by default, even if the data you provided is normalised. So, we need to have control on that level.

Reply:
To be fair, I think just reminding the user would suffice.